taskiq/abc/broker.py

```diff
@@ -512,12 +513,19 @@ def _register_task(
         task: AsyncTaskiqDecoratedTask[Any, Any],
     ) -> None:
         """
-        Mehtod is used to register tasks.
+        Method is used to register tasks.
 
         By default we register tasks in local task registry.
         But this behaviour can be changed in subclasses.
 
+        This method may raise TaskRejectedError if task has already been
+        registered to a different broker.
+
         :param task_name: Name of a task.
         :param task: Decorated task.
         """
+        if task.broker != self:
+            raise TaskRejectedError(
```

Comment:
It's not a syntax error. It's [izulu](https://github.com/pyctrl/izulu). We use it for errors. 

Please define separate error class for this error with 

__template__ and broker argument.

Reply:
rgr. This is the syntax error:

![image](https://github.com/user-attachments/assets/6cef96b5-da38-447d-898f-b6de8bc2f341)

![image](https://github.com/user-attachments/assets/0832d492-c3ad-4783-90f7-1e704193822c)